Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -114,16 +114,24 @@ namespace SandboxEditor
             // Cache the ray trace results when doing manipulator interaction checks, no need to recalculate after
             if (event.m_priority == ManipulatorPriority)
             {
+
                 const auto* position = event.m_inputChannel.GetCustomData<AzFramework::InputChannel::PositionData2D>();
                 AZ_Assert(position, "Expected PositionData2D but found nullptr");
 
+                if(m_captureStart) {
+                    m_virtualPosition = position->m_normalizedPosition;
+                    m_captureStart = false;
+                } else {
+                    m_virtualPosition += position->m_normalizedPositionDelta;
+                }
```

Comment:
standards: brace positions and spacing around the `if` and parens (`clang-format` should sort this out)